packages/main/src/components/top-navigation-bar/TopNavigationBar.module.css

```diff
@@ -83,9 +83,6 @@
   height: 25px;
   border-radius: 50%;
 }
-.world:hover {

```

Comment:
You've just done away with the code. So how will a user know if they've hovered on the icon?
I suggest you add a better hover effect, not just remove the code entirely.